packages/storybook-addon/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This should be ignored and copied by the `prepare` script.
You can probably just remove this file from the PR and the gitignore should prevent it from being committed again